<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTest</class>
 <widget class="QMainWindow" name="ClientTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="ConnectionBorder">
      <item>
       <widget class="QLineEdit" name="inputAdress">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>2222</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonConnectServer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="ShowMessages"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="MessageBorder">
      <item>
       <widget class="QLineEdit" name="inputMessage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendMessage">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Отправить сообщение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="sendImage">
      <property name="text">
       <string>Отправить изображение</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
